test/cpp/interop/interop_client.h

```diff
@@ -109,8 +121,9 @@ class InteropClient {
    public:
     // If new_stub_every_call = true, pointer to a new instance of
     // TestServce::Stub is returned by Get() everytime it is called
-    ServiceStub(ChannelCreationFunc channel_creation_func,
-                bool new_stub_every_call);
+    ServiceStub(
+        std::function<std::shared_ptr<grpc::Channel>()> channel_creation_func,
```

Comment:
Shouldn't this still use `ChannelCreationFunc` rather than spelling out the type?

Reply:
Looks like you made this change below but not here in the argument to the `ServiceStub` ctor.

Reply:
Done